cpp/dolfinx/mesh/cell_types.cpp

```diff
@@ -313,17 +123,18 @@ mesh::get_sub_entities(CellType type, int dim0, int dim1)
     throw std::runtime_error(
         "mesh::get_sub_entities supports getting edges (d=1) at present.");
   }
+  // TODO: get this data from basix
```

Comment:
Should this be done before a merge?

Reply:
I've made an issue for this one (https://github.com/FEniCS/dolfinx/issues/1338). Not done yet as I think this data is not immediately available in basix yet